tests/ffi/dev-mem_not_present/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+. ../../e2e/lib/utils
+
+check_dev-mem_not_present(){
+    expected_result="ls: cannot access '/dev/mem': No such file or directory"
+    return_from_cmd=$(podman exec -t qm bash -c "ls -l /dev/mem")
+
+    # Check /dev/mem is not present in QM partition
+    if [[ $? != "0"  && "${return_from_cmd}" == "${expected_result}"* ]]; then
```

Comment:
question, do you need more than $? != "0" ?

Reply:
Yes, it is redundant, I will improve it.

Reply:
done